tests.py

```diff
@@ -546,11 +535,9 @@ def test_check_sms_state_error_network(self, mock_post):
 
         mock_post.assert_called_once_with(
             sms_plusserver.SMSService.SMS_STATE_URL,
-            {
-                'handle': 'd41d8cd98f00b204e9800998ecf8427e',
-            },
+            {'handle': 'd41d8cd98f00b204e9800998ecf8427e',},
```

Comment:
Please reconfigure your Black so it wouldn't leave the trailing commas on single-line literals.

Reply:
- [ ] I will remove  trailing commas

Reply:
Done with https://github.com/W-Z-FinTech-GmbH/sms_plusserver/pull/6/commits/3f724767adb145aac9d5fcc842fea765ed9d774b